Doc/library/subprocess.rst

```diff
@@ -52,10 +52,11 @@ underlying :class:`Popen` interface can be used directly.
 
    If *capture_output* is true, stdout and stderr will be captured.
    When used, the internal :class:`Popen` object is automatically created with
-   ``stdout=PIPE`` and ``stderr=PIPE``. The *stdout* and *stderr* arguments may
-   not be supplied at the same time as *capture_output*.  If you wish to capture
-   and combine both streams into one, use ``stdout=PIPE`` and ``stderr=STDOUT``
-   instead of *capture_output*.
+   ``stdout=subprocess.PIPE`` and ``stderr=subprocess.PIPE``. The *stdout* and
+   *stderr* arguments may not be supplied at the same time as *capture_output*.
+   If you wish to capture and combine both streams into one, use
+   ``stdout=subprocess.PIPE`` and ``stderr=subprocess.STDOUT`` instead of
+   *capture_output*.
```

Comment:
How about this?

```suggestion
   *stdout* and *stdin* both set to :data:`~subprocess.PIPE`.
   The *stdout* and *stderr* arguments may not be supplied at the same time as *capture_output*.
   If you wish to capture and combine both streams into one,
   set *stdout* to :data:`~subprocess.PIPE`
   and *stderr* to :data:`~subprocess.STDOUT`,
   instead of using *capture_output*.
```